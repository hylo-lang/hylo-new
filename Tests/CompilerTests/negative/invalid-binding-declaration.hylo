//! no-std

struct A {}

fun g(a: A) {
  let (x, y): A = a
  let (z) = (a)
}
