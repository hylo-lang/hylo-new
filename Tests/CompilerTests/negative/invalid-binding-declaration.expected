negative/invalid-binding-declaration.hylo:6.7-13: error: tuple pattern cannot match values of type 'A'
  let (x, y): A = a
      ~~~~~~
